### DISCUSSION TITLE
Backup>Restore database from UI?

### DISCUSSION THREAD

**Discussion by Dievale** [immich-app/immich#14967] [Q&A]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Having an automatic .sql backup is helpful but useless for a rookie user like me and many. Please add backup/restore from UI if possible.
I installed immich in truenass scale apps following this:
https://immich.app/docs/install/truenas/
Then I tried to manually back up/restore following this:
https://immich.app/docs/administration/backup-and-restore#manual-backup-and-restore
When I open shell in truenass and type 
docker exec -t immich_postgres pg_dumpall --clean --if-exists --username=postgres | gzip > "/mnt/Seagate 5 TB/immich/backups/immich-db-TEST.sql.gz" to backup I got:
root@truenas[~]# docker exec -t immich_postgres pg_dumpall --clean --if-exists --username=postgres | gzip > "/mnt/Seagate 5 TB/immich/backups/immich-db-TEST1.sql.gz"
Error response from daemon: No such container: immich_postgres

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by danieldietzler** [comment]
Mixing up a feature request and asking for help is kind of confusing tbh.
In order to get help with your actual question we'd need more information, specificaly truenas & Immich version

**Reply by Dievale** [reply]
Thank for the prompt reply and sorry if I did the request wrong
Truenas ElectricEel-24.10.1
Immich the latest version Name:
[immich](http://192.168.77.200/ui/apps/available/community/immich)
App Version:
v1.123.0
Version:
1.7.15
From community train

**Reply by danieldietzler** [reply]
Can you run `docker ps`?

**Reply by Nordlicht-13** [reply]
docker ps is working on TrueNAS-Shell or via ssh.

**Reply by danieldietzler** [reply]
Huh? Do you have the same issue? Why did you respond to this? @Nordlicht-13

**Reply by Nordlicht-13** [reply]
Same issu, immich won't start any more.
With fresh DB immich is starting. I want to restore my backup and don't know how it is done under TrueNAS or dockge.

**Reply by danieldietzler** [reply]
Please open a new discussion then or join our discord (https://discord..immich.app). And share all the information you've got
